MovingBlocks/DestinationSol
# Money Over 9999 Display Issue #136

# Description
* Attempted to fix Issue #136
* Created a `drawStringLeftAligned` method to left align text.

I'm not sure if this was the correct way to do this. But from what I could tell the only options were to center or not center. The previous code was shifting the centered text off to the left. Since the text was drawn from the center of the string, when the money string got long enough it would draw over the money icon.

# Testing
* See issue #136 to reproduce the draw over.
* Edit your prevShip.ini, change the money value to something over 9999. It should no longer render over the icon, and remain left aligned.
* Change it to the max value 2147483647, it should still not draw over the icon.

# Notes
I ran the default Intellij's code formatter. I have my tabs set to 4 spaces. You don't have any documentation on code format. I see 2 space tabs, and I think I see 4 space ones as well. let me know which one you prefer. 

### Pre Pull Request Checklist:

- [x] Code has been scanned with [SonarLint](http://www.sonarlint.org/intellij/)
- [ ] Methods have been annotated with @ Nullable and @ Nonnull ([Read more](https://github.com/MovingBlocks/DestinationSol/wiki/@Nullable-&-@Nonnull-Quickstart))
- [x] There are no errors present in the project
- [x] Code has been formatted and indented
- [ ] Methods have appropriate Javadoc ([How to write Javadoc](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html))
